src/getting-started/create-bootable-media/bootable-usb-using-windows.md

```diff
@@ -1 +1,101 @@
-Users creating installation media from Windows can use Rufus. Rufus is an open source startup disk creator capable of writing ISO images to portable storage media.
\ No newline at end of file
+# Create a Bootable Pop!\_OS USB in Windows 10
+
+This section explains the process of creating a bootable Pop!_OS USB in macOS using balena Etcher.
+
+- [Install ISO flashing software.](/Getting-Started/Create-bootable-media/bootable-usb-using-windows.html#install-etcher)
+- [Download the Pop!\_OS.iso file.](/Getting-Started/Create-bootable-media/bootable-usb-using-windows.html#download-the-pop_osiso-file)
+- [Verify the Pop!\_OS.iso Hash Value](/Getting-Started/Create-bootable-media/bootable-usb-using-windows.html#verify-the-hash-value)
```

Comment:
The other three table of contents links are written as imperative sentences; this one's written in title case with no punctuation. It should be consistent.